File: test/unit/code_parser/whole_tests/views/portal_2011.cs
// Copyright (c) 2000 ActivePDF, Inc.
// ActivePDF Portal 2011
// Example generated 01/01/2000 

using System;
using System.Diagnostics;
using System.IO;

partial class Example : System.Web.UI.Page
{
    protected void Page_Load(object sender, EventArgs e)
    {
    	if (!IsPostBack)
    	{
    		// Specify the document to open with portal
    		this.PdfWebControl1.CreateDocument("filename", System.IO.File.ReadAllBytes(Server.MapPath(@"portal.pdf")));
    	}
    }
    
    protected void SaveComplete(object sender, SaveCompleteEventArgs e)
    {
    	// Check the SaveType to determine action to perform
    	switch (e.SaveType)
    	{
    		// Save the file to disk when saving or downloading
    		case SaveType.Save:
    		case SaveType.Download:
    
    			break;
    
    		// Ignore all other save types
    		default:
    
    			break;
    	}
    }
  
}

File: test/unit/code_parser/whole_tests/views/portal_2011.vb
' Copyright (c) 2000 ActivePDF, Inc.
' ActivePDF Portal 2011
' Example generated 01/01/2000 

Imports System.IO

Partial Class Example
  Inherits System.Web.UI.Page
    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
    
    	If Not IsPostBack Then
    
    		' Specify the document to open with portal
    		Me.PdfWebControl1.CreateDocument("filename", System.IO.File.ReadAllBytes(Server.MapPath("portal.pdf")))
    
    	End If
    End Sub
    
    Protected Sub SaveComplete(ByVal sender As Object, ByVal e As SaveCompleteEventArgs)
    
    	' Check the SaveType to determine action to perform
    	Select Case e.SaveType
    
    		' Save the file to disk when saving or downloading
    		Case SaveType.Save, SaveType.Download
    
    			Exit Select
    
    		' Ignore all other save types
    		Case Else
    
    			Exit Select
    	End Select
    End Sub
  
End Class